### PR TITLE
DM-45131: Remove unnecessary lsst.log imports

### PR DESCRIPTION
**Pull request by timj**

### CODE REVIEW

File: tests/test_opt.py [hunk 1]
```diff
@@ -22,15 +22,15 @@
 #
 import unittest
 import numpy
+import logging
```
Comment (mwittgen) [inline tests/test_opt.py:25]: Why  using `logging` here and not replacing all occurrences in `tests`?

**Review by mwittgen — APPROVED** [review]
I guess the ticket does not intend to switch completely to `logging` and remove the dependency to `lsst.log`

File: tests/test_opt.py (hunk 1 above)
Comment (timj) [reply tests/test_opt.py:25]: What other occurrences? The other test files imported `lsst.log` but didn't use it. This test does write log messages so needed a logger.

**Comment by mwittgen** [comment]
Sorry wrong review.

**Review by mwittgen — APPROVED** [review]

**Pull request merged**